sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A tool for recovering MBR partitions tables in hard disk"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are\
+on a PC type, MBR-partitioned hard disk in case the primary partition table was damaged."
```

Comment:
```diff
-DESCRIPTION="Gpart is a small tool which tries to guess what partitions are\
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
-on a PC type, MBR-partitioned hard disk in case the primary partition table was damaged."
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
```